apps/website-25/src/pages/certification.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  CTALinkOrButton,
+  H1,
+  P,
+  ProgressDots,
+  Section,
+  Footer,
+  ShareButton,
+} from '@bluedot/ui';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import useAxios from 'axios-hooks';
+import { FaCircleCheck } from 'react-icons/fa6';
+import { GetCertificateResponse } from './api/certificates/[certificateId]';
+import { ROUTES } from '../lib/routes';
+
+const CertificatePage = () => {
+  const router = useRouter();
+  const { id, r } = router.query;
+  const certificateId = id ?? r;
+
+  if (!certificateId) {
+    return (
+      <Section>
+        <H1>Invalid certificate</H1>
```

Comment:
```suggestion
        <h1>Invalid certificate</h1>
```
these styles are now directly consumable via Tailwind base styles instead of separately maintained as a UI component. ill follow up with the UI design discussion Jan and I had in a thread somewhere